code/script.js

```diff
@@ -29,19 +59,643 @@ const showMessage = (message, sender) => {
+    `
+
+    const yesBtn = document.getElementById('yes')
+    const noBtn = document.getElementById('no')
+  
+    yesBtn.addEventListener('click', () => {
+      showMessage(`Yes!`, 'user')
+      ending()
+    })
+
+    yesBtn.onclick = function () {
+      if (yesBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+    noBtn.onclick = function () {
+      if (noBtn.style.display !== 'none') {
+        yesBtn.style.display = 'none';
+        noBtn.style.display = 'none';
+      };
+    }
+
+    noBtn.addEventListener('click', () => {
+      input.value=''
+      setTimeout (() => showMessage(`Too bad! I will make you start over`, 'bot'),1000);
+      setTimeout(() => beginAgain(), 5000);
+    })
+  }
+ })
+
+  } else { //if the bot does not recognize the mood
+    showMessage(`I don't recognize that mood..`,'bot')
+    //dropdown with all the moods
+    inputWrapper.innerHTML = `
+    <select id="select5">
+     <option value="" selected disabled> Select a mood..</option>
+     <option value="happy">happy</option>
+     <option value="glad">glad</option>
+     <option value="excited">excited</option>
+     <option value="angry">angry</option>
+     <option value="mad">mad</option>
+     <option value="frustrated">fustrated</option>
+     <option value="irritated">irritated</option>
+     <option value="sad">sad</option>
+     <option value="depressed">depressed</option>
+     <option value="blue">blue</option>
+     <option value="in-love">in love</option>
+     <option value="love">love</option>
+     <option value="romantic">romantic</option>
+   </select>
+   <button id="rerun"> Go </button>
+   `
+//human can chose a mood in the dropdown and press go to listen
+   document.getElementById('rerun').onclick = function() {
+     console.log('now I have clicked', document.getElementById('select5').value);
+     songSelection(document.getElementById('select5').value)
+   }
+  
+  }
+
 }
+//bot ending question
+    const ending = () => {
+      questionNumber = 6
+      showMessage(`Thank you for chatting with me!`, 'bot')
+    }
+//reloads the page when pressing no when listening to music
+    const beginAgain = () => {
+      questionNumber = 7
+      location.reload();
+    }
+    
+//Logic that sends the human forward in the chat
+const handleInput = (event) => {
+  event.preventDefault()
+  console.log(questionNumber)
+  if (questionNumber === 1) {
+    handleNameInput()
+  }
+  else if (questionNumber === 2) {
+    console.log(2)
+    setTimeout(() => whatMood(event), 1000);
+  } else if (questionNumber === 3) {
+    console.log(3);
+    setTimeout(() => moodAnswer(event), 1000);
+  } else if (questionNumber === 4) {
+    setTimeout(() => songSelection(event), 1000);
+  } else if (questionNumber === 6) {
+    setTimeout(() => ending(event), 5000);
+  } else { (questionNumber === 7) 
+    setTimeout (() => beginAgain(), 1000); 
+  }
+  }
```

Comment:
Soo cool that you had the time to structure your code like this and not just by send one answer to the next question. GOOD job!!